pkg/account/domain/account.go

```diff
@@ -118,23 +118,27 @@ func (a *AccountV2) AddSearchFilter(
 	name string,
 	query string,
 	targetType proto.FilterTargetType,
-	environmentID string, defaultFilter bool) error {
+	environmentId string, defaultFilter bool) (*proto.SearchFilter, error) {
```

Comment:
In Go, the id is in upper case.

```suggestion
	environmentID string, defaultFilter bool) (*proto.SearchFilter, error) {
```